web-registry/src/lib/ecocredit/api.ts

```diff
@@ -163,6 +182,32 @@ export const getBatchWithSupplyForDenom = async (
   }
 };
 
+export const getReadableMessages = (txResponse: TxResponse): string => {
```

Comment:
multiple messages can be included in a tx so I guess it should be return string[] instead